recipes/llvm-openmp/all/conanfile.py

```diff
@@ -88,11 +92,39 @@ def package(self):
         self.copy(pattern="LICENSE.txt", src=self._source_subfolder, dst="licenses")
         cmake = self._configure_cmake()
         cmake.install()
+        self._create_cmake_module_alias_targets(
+            os.path.join(self.package_folder, self._module_file_rel_path),
+            {
+                "OpenMP::OpenMP_C": "OpenMP::OpenMP",
+                "OpenMP::OpenMP_CXX": "OpenMP::OpenMP"
+            }
+        )
+
+    @staticmethod
+    def _create_cmake_module_alias_targets(module_file, targets):
+        content = ""
+        for alias, aliased in targets.items():
+            content += textwrap.dedent("""\
+                if(TARGET {aliased} AND NOT TARGET {alias})
+                    add_library({alias} INTERFACE IMPORTED)
+                    set_property(TARGET {alias} PROPERTY INTERFACE_LINK_LIBRARIES {aliased})
+                endif()
+            """.format(alias=alias, aliased=aliased))
+        tools.save(module_file, content)
+
+    @property
+    def _module_file_rel_path(self):
+        return os.path.join("lib", "cmake", "conan-official-{}-targets.cmake".format(self.name))
 
     def package_info(self):
+        self.cpp_info.set_property("cmake_file_name", "OpenMP")
+        self.cpp_info.set_property("cmake_target_name", "OpenMP::OpenMP")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "OpenMP::OpenMP")
        self.cpp_info.set_property("cmake_target_aliases", ["OpenMP_C", "OpenMP_CXX"])
```
https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmakedeps.html#properties

Reply:
Thank you for reviewing.
I will add a alias property and some comments.